docs/_sidebar.md

```diff
@@ -20,6 +20,7 @@
 * Grading
   * [Overview](grading/README.md)
   * [Rubric](grading/rubric.md)
+  * [Excused Absences](grading/excused_absences.md)
```

Comment:
We might change this to `grading/attendance.md` to encapsulate all attendance-related grading policies - including but not limited to Excused Absences. No need to change it here - just a comment :)

Reply:
Ok, that makes a lot more sense